sk_sidekick/lib/src/release/core_bundled_version_bump.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:pub_semver/pub_semver.dart';
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:sk_sidekick/sk_sidekick.dart';
+
+/// Updates the version in sidekick_core/lib/sidekick_core.dart
+void coreBundledVersionBump(
+  DartPackage package,
+  Version oldVersion,
+  Version newVersion,
+) {
+  if (package != skProject.sidekickCorePackage) {
```

Comment:
Comparison fails on the current commit because `DartPackage.operator ==` doesn't canonicalize `root.path` (ends up comparing `'sidekick_core' == '/Users/pepe/repos/sidekick/sidekick_core'`), but should work when the changes in #118 are merged :)